common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -11,13 +11,13 @@
   </ItemGroup>
   <ItemGroup>
     <!-- Add an OverrideDailyVersion attribute to prevent the Update-Dependencies script from overwriting it with a daily build version -->
-    <PackageReference Include="Azure.Identity" Version="1.5.0-beta.2" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.5.0" />
-    <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.5.0" />
-    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.2.0" />
+    <PackageReference Include="Azure.Identity" Version="1.5.0-beta.3" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.6.0" />
+    <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.6.0" />
+    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.3.0-beta.1" />
```

Comment:
Do we really want betas?

Reply:
It's typically been the latest published.   As the package owner, what's your recommendation for Key Vault?

Reply:
I guess since Identity is doing it, I don't want to feel left out.